pyperformance/compile.py

```diff
@@ -300,9 +300,9 @@ def compile(self):
 
         if self.conf.pgo:
             # FIXME: use taskset (isolated CPUs) for PGO?
-            self.run('make', 'profile-opt')
+            self.run('make', 'profile-opt', '-j%d' % self.conf.jobs)
         else:
-            self.run('make')
+            self.run('make', '-j%d' % self.conf.jobs)
```

Comment:
It may make sense to only add the flag if the config option is set:

```python
        argv = ['make']
        if self.conf.pgo:
            # FIXME: use taskset (isolated CPUs) for PGO?
            argv.append('profile-opt')
        if self.conf.jobs:
            argv.append('-j%d' % self.conf.jobs)
        self.run(*argv)
```

So the config default would be `None` instead of 1.

The reason to do this is: the current default could be something other than 1 given that there are other ways to specify the number of concurrent jobs to run.

Reply:
Should we apply the `-j` parameters with the pgo job?

Reply:
It would be.  That's `if self.conf.jobs:` not `elif self.conf.jobs:`.  Perhaps I misunderstood your question?

Reply:
Oh, I understood! Updated.